runtime/browser/xwalk_content_base.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#ifndef XWALK_RUNTIME_BROWSER_XWALK_CONTENT_BASE_H_
+#define XWALK_RUNTIME_BROWSER_XWALK_CONTENT_BASE_H_
+
+#include "base/memory/scoped_ptr.h"
+#include "base/prefs/pref_change_registrar.h"
+
+namespace content {
+class WebContents;
+}
+
+namespace xwalk {
+
+class XWalkContent;
+
+class XWalkContentBase {
+ public:
+  explicit XWalkContentBase(scoped_ptr<content::WebContents> web_contents);
+  virtual ~XWalkContentBase();
+  void InitAutofillIfNecessary(bool enabled);
+  void CreateUserPrefServiceIfNecessary();
+  void SetXWalkUserData();
```

Comment:
As I see the methods above are not called outside the class, they should not be public